src/domain/journey/space/createSpace/CreateSpaceDialog.tsx

```diff
@@ -41,8 +40,7 @@ interface FormValues extends SpaceEditFormValuesType {
 const CreateSpaceDialog = () => {
   const handleClose = useBackToStaticPath(ROUTE_HOME);
```

Comment:
If you close the dialog on user->account, it lands you back on the dashboard. Ideally, it should either accept a path or just close itself without redirection.

Reply:
I'd like to keep them separated in case their visualizations divert more, and together with their non-skeleton card... But I have done now the InnovationPack skeleton a bit different to have the excuse :D

Reply:
This was a bad copy&paste, good catch